wallet/wallet.go

```diff
@@ -731,7 +731,9 @@ func (w *Wallet) fetchHeaders(chainClient *chain.RPCClient) (int, error) {
 // in the main chain.  This is the block a rescan should begin at (inclusive),
 // and is only relevant when the number of fetched headers is not zero.
 func (w *Wallet) FetchHeaders(chainClient *chain.RPCClient) (count int,
-	rescanFrom chainhash.Hash, rescanFromHeight int32, err error) {
+	rescanFrom chainhash.Hash, rescanFromHeight int32,
+	mainChainTipBlockHash chainhash.Hash, mainChainTipBlockHeight int32,
+	err error) {
```

Comment:
Please make this fit on 2 lines even if it goes over 80 cols, I think that still reads better (especially in godoc).